modules/serverless-service-mesh-enable-sidecar-injection.adoc

```diff
@@ -5,13 +5,18 @@
 [id="serverless-enable-sidecar_{context}"]
 = Enabling sidecar injection for a Knative service
 
-You can add an annotation to the Service resource YAML file to enable sidecar injection for a Knative service.
+You can add the `sidecar.istio.io/inject="true"` annotation to a Knative service to enable sidecar injection that service.
```

Comment:
...sidecar injection *in* that service? Maybe "with".  Need a preposition, right?  Maybe "for that service."